Source/ApiTemplate/build.cake

```diff
@@ -89,7 +89,10 @@ Task("DockerBuild")
     .Description("Builds a Docker image.")
     .DoesForEach(GetFiles("./**/Dockerfile"), dockerfile =>
```

Comment:
It might be better if we exclude the `.devcontainer` directory on this line either using globbing syntax or a chained call.

Reply:
Ohh yes I agree, I didn't know there was a way to do that.